app/design-system/design-system-hedvig/src/main/kotlin/com/hedvig/android/design/system/hedvig/Stepper.kt

```diff
@@ -0,0 +1,693 @@
+      launch { labelColor.animateTo(initialLabelColor, animationSpec = enterAnimationSpec) }
+      launch { textColor.animateTo(initialTextColor, animationSpec = enterAnimationSpec) }
+    }
+  }
+  when (stepperStyle) {
+    Default -> DefaultStepper(
+      stepperSize = stepperSize,
+      text = text,
+      onMinusClick = onMinusClick,
+      onPlusClick = onPlusClick,
+      containerColor = containerColor.value,
+      textColor = textColor.value,
+      plusColor = plusColor.value,
+      minusColor = minusColor.value,
+      errorText = errorText,
+      showError = showError,
+      modifier = modifier,
+    )
+
+    is Labeled -> LabeledStepper(
+      stepperSize = stepperSize,
+      text = text,
+      labelText = stepperStyle.labelText,
+      onMinusClick = onMinusClick,
+      onPlusClick = onPlusClick,
+      containerColor = containerColor.value,
+      textColor = textColor.value,
+      labelColor = labelColor.value,
+      plusColor = plusColor.value,
+      minusColor = minusColor.value,
+      errorText = errorText,
+      showError = showError,
+      modifier = modifier,
+    )
+  }
+}
+
+@Composable
+private fun DefaultStepper(
+  stepperSize: StepperDefaults.StepperSize,
+  text: String,
+  onMinusClick: () -> Unit,
+  onPlusClick: () -> Unit,
+  containerColor: Color,
+  textColor: Color,
+  plusColor: Color,
+  minusColor: Color,
+  showError: Boolean,
+  errorText: String?,
+  modifier: Modifier = Modifier,
+) {
+  Column(modifier) {
+    Surface(
+      shape = stepperSize.size.shape,
+      color = containerColor,
+      modifier = Modifier.defaultMinSize(minHeight = stepperSize.size.minHeight(Default)),
+    ) {
+      Row(
+        Modifier.padding(stepperSize.size.contentPadding(Default)),
+        verticalAlignment = Alignment.CenterVertically,
+      ) {
+        HedvigText(
+          text = text,
+          style = stepperSize.size.textStyle(Default),
+          color = textColor,
+        )
+        Spacer(Modifier.weight(1f))
+        Spacer(Modifier.width(4.dp))
+        StepperSymbols(
+          onMinusClick,
+          onPlusClick,
+          plusColor,
+          minusColor,
+          modifier = Modifier.padding(stepperSize.size.symbolsPadding),
+        )
+      }
+    }
+    if (showError && errorText != null) {
+      HedvigText(
+        text = errorText,
+        color = stepperColors.errorTextColor,
+        style = stepperSize.size.labelTextStyle,
+        modifier = Modifier.padding(stepperSize.size.errorTextPadding(Default)),
+      )
+    }
+  }
+}
+
+@Composable
+private fun StepperSymbols(
+  onMinusClick: () -> Unit,
+  onPlusClick: () -> Unit,
+  plusColor: Color,
+  minusColor: Color,
+  modifier: Modifier = Modifier,
+) {
+  Row(modifier) {
+    IconButton(onClick = onMinusClick) {
+      Icon(HedvigIcons.Minus, null, tint = minusColor)
+    }
+    IconButton(onClick = onPlusClick) {
+      Icon(HedvigIcons.Plus, null, tint = plusColor)
+    }
+  }
+}
```

Comment:
These in figma seem to be like this:
24x24icon - 16space - 24x24icon

I think with this impl we get (since the min touch target is making them 48.dp)
48x48 - 48x48,
Where if we have the icons be 24x24, it ends up having 12 and 12 spacing on each side, so the end result is
24x24 - 12 - 12 - 24x24
So our space inbetween ends up being 24dp instead of 16 as Figma describes.

Could we perhaps be explicit about the inner icons being 24.dp, so like
```
IconButton(onClick = onPlusClick) {
  Icon(HedvigIcons.Plus, null, tint = plusColor, modifier = Modifier.size(24.dp))
}
```
And then see if we can place them closer to each other if my assumption above is true?
Perhaps with the negative `Arrangement.spacedBy` as I mention in another comment, not sure.